src/Runner/Runner.php

```diff
@@ -169,9 +169,9 @@ private function fixFile(\SplFileInfo $file, LintingResultInterface $lintingResu
                 $fixer->fix($file, $tokens);
 
                 if ($tokens->isChanged()) {
+                    $appliedFixers[$fixer->getName()] = $tokens->findChangedLines();
```

Comment:
Isn't that exactly what I wrote in the issue summary?

>:warning: Breaks BC for modified formatters because appliedFixers structure was changed from a list of strings to a map of integers (line numbers) with string keys (fixer names).